packages/interface/src/features/home/components/Glossary.tsx

```diff
@@ -0,0 +1,71 @@
+import { Heading } from "~/components/ui/Heading";
+
+import { FAQItem } from "./FaqItem";
+
+export const Glossary = (): JSX.Element => (
+  <div className="mt-14 flex flex-col items-center justify-center sm:mt-28 dark:text-white" id="Glossary">
```

Comment:
After clicking the `faq` or `glossary` on the help dropdown menu, the href = page url changed, however, since it's the same page without refreshing / detecting any difference, it won't scroll to the right block, I can only see it's on the right block until I refresh the page.

Thinking about this: https://stackoverflow.com/questions/75490284/nextjs-13-experimental-app-dir-hash-in-routes-not-directing-to-the-hash-id
(detecting hash changed -> scroll to view)

Reply:
Thanks! after trying for a while, realized that the one causing the bug was the dropdown menu, so added a function to close it when link is clicked.